### DISCUSSION TITLE
Is old quasar frameworkd forum removed?

### DISCUSSION THREAD

**Discussion by quroom** [quasarframework/quasar#16341] [General - Components / Directives / etc] [answered]
It has lots of questions and answers.
It there any way to read that Q&A?

https://github.com/quasarframework/quasar/discussions/topic/3698/q-dialog-dragging-and-scaling-resizing

It can be searched in google. but the page is broken.

**Comment by rstoenescu** [comment, marked as answer]
Hi.

Unfortunately, the old forum has been removed. We're focusing our resources (time & money) on a better server for the docs, so we cut from the very very old services (like the old forum). Sorry for the inconvenience, but it is what it is. Maintaining that old server required a ridiculous amount of time.

**Reply by quroom** [reply]
Thanks for answering.
I left for someone facing same issue.
You can access removed page with webcache.
